homeassistant/components/edl21/sensor.py

```diff
@@ -52,92 +62,196 @@ class EDL21:
     """EDL21 handles telegrams sent by a compatible smart meter."""
 
     # OBIS format: A-B:C.D.E*F
-    _OBIS_NAMES = {
+    _OBIS_SENSOR_ENTITY_DESCRIPTIONS = {
```

Comment:
Move the key (OBIS value) of this dictionary to the `key` field of the entity description (that is the purpose of the key field of the entity description).